_data/changelogs/component-combo-box.yml

```
@@ -2,7 +2,15 @@ title: Combo box
 type: component
 changelogURL:
 items:
-  - date: 2024-03-13
+  - date: NNNN-NN-NN
+    summary: Removed verbose aria instructions.
+    summaryAdditional: Combobox now only reads default screenreader instructions.
```

This feels like it could be _slightly_ misleading since the combo box will read out additional information like the label and other semantic info. 

**Suggestion (non-blocking):** 

```suggestion
    summaryAdditional: Combo box now relies on the default instructions provided by screen readers.
```